docs/tutorials/openshift.md

```diff
@@ -25,7 +25,7 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:latest
+        image: docker.io/porscheinformatik/external-dns:latest
```

Comment:
I understand. But wouldn't the kubernetes-staging image be better, since as long as no new release is released new users would try an image which doesn't reflect the new capability.